### ISSUE TITLE
force itself on top off steam's shameless keyboard hook.

### ISSUE THREAD

**Issue by paxcoder** [chigkim/VOCR#21]
please, try to make it prioritize itself instead off steam, and make it OCR steam windows

**Comment by chigkim** [comment]
I have no idea what you're referring to. Can you rephrase your question?
Unfortunately, I don't have access to steam, so I can't help much.

**Comment by paxcoder** [comment]
I mean do you can force the apps shortcut work on steam window? for now ocring the steam window doesnt work.

**Comment by chigkim** [comment]
Ah, if Steam is swallowing the shortcuts before getting to VOCR, there's nothing I can do about it.

**Comment by chigkim** [comment]
As an alternative, you can go to your desktop and scan the entire screen, and you should be able to see Steam as well.
